benchmarks/inc/udt.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+
+template <typename Contained>
+struct aggregate {
+    Contained c;
+
+    friend bool operator==(const aggregate& lhs, const aggregate& rhs) = default;

```

Comment:
Suggestion: In product code, we conventionally omit parameter names for defaulted functions (the rationale being that the defaulted behavior is "well-known", so parameter names don't add value). In test code we're less rigorous about matching product conventions, so if you want to keep this as-is you can, but I'd recommend omitting the names. Occurs below.